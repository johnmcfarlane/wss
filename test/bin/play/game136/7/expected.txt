 54  1,7  | HEIAU
 39  1,9  | EAU
 39  1,9  | IOU
 32  4,4  | PIXIE
 30  1,7  | HAU
 29  1,6  | OHIA
 27  1,11 - UTAI
 24  1,7  | HAE
 24  1,7  | HAO
 24  1,7  | HEE
 24  1,7  | HEI
 24  1,7  | HIA
 24  1,7  | HIE
 24  1,7  | HOE
 24  1,7  | HOI
 24  1,11 - UTA
 24  1,11 - UTE
 24  1,11 - UTI
 22  1,8  - EDEA
 22  1,8  - IDEA
 22  1,8  - IDEE
 22  1,8  - IDEO
 22  3,7  | JOIE
 22  1,8  - ODEA
 21  1,11 - ETUA
 21  1,11 - ETUI
 20  2,8  - DAU
 19  1,8  - ADO
 19  1,8  - EDO
 19  1,8  - IDE
 19  3,7  | JAI
 19  1,8  - ODA
 19  1,8  - ODE
 18  2,8  - DEIA
 18  1,9  - ESAU
 18  1,9  - ESU
 18  1,11 - ITEA
 18  1,9  - USEE
 17  2,8  - DAO
 17  2,8  - DEA
 17  2,8  - DEE
 17  2,8  - DEI
 17  2,8  - DOA
 17  2,8  - DOE
 15  1,11 - ATE
 15  1,11 - ATI
 15  1,11 - ETA
 15  1,11 - ITA
 15  2,11 - TIOU
 15  1,9  - USE
 14  1,9  | AE
 14  2,8  - DA
 14  2,8  - DE
 14  2,8  - DO
 14  1,9  | EE
 14  3,10 | EE
 14  1,9  | EO
 14  3,10 | EO
 14  1,9  | IO
 14  3,7  | JA
 14  3,7  | JE
 14  3,7  | JO
 14  1,9  | OE
 14  3,10 | OE
 14  3,10 | OI
 12  1,9  - ESE
 12  1,9  - ISE
 12  1,9  - OSE
 12  2,11 - TAU
 12  2,11 - TIAO
 12  2,11 - TIU
 12  2,11 - TOEA
 12  2,11 - TOU
 12  2,11 - TUA
 12  2,11 - TUE
 12  2,11 - TUI
 11  6,5  - VEAU
 10  1,10 | AE
 10  1,6  | AHI
 10  1,6  | AHO
 10  1,10 | AI
 10  1,10 | EE
 10  1,10 | OE
 10  1,10 | OI
  9  5,3  - MYA
  9  2,11 - TAE
  9  2,11 - TAI
  9  2,11 - TAO
  9  2,11 - TEA
  9  2,11 - TEE
  9  2,11 - TIE
  9  2,11 - TOA
  9  2,11 - TOE
  9  2,11 - TOI
  9  1,9  - US
  8  5,7  | EAU
  8  7,7  | EAU
  8  3,9  | IO
  8  7,6  | IOU
  8  3,9  | OE
  8  6,5  - VAU
  7  1,8  - AD
  7  1,8  - ED
  7  1,7  | HA
  7  1,7  | HE
  7  1,7  | HI
  7  1,7  | HO
  7  1,8  - ID
  7  1,8  - OD
  7  5,7  | OUI
  7  7,7  | OUI
  7  1,11 - UT
  7  6,5  - VAI
  7  6,5  - VEE
  7  6,5  - VEI
  7  6,5  - VIA
  7  6,5  - VIE
  7  6,5  - VOE
  6  1,6  - AE
  6  7,6  | AE
  6  1,6  | AH
  6  1,9  - AS
  6  1,6  - EE
  6  7,6  | EE
  6  1,6  | EH
  6  7,6  | EO
  6  1,9  - ES
  6  7,6  | IO
  6  1,9  - IS
  6  1,6  - OE
  6  1,6  | OH
  6  1,9  - OS
  6  2,9  - SUE
  6  2,9  - SUI
  6  2,11 - TA
  6  2,11 - TE
  6  2,11 - TI
  6  2,11 - TO
  5  4,1  | AL
  5  4,1  | EL
  5  2,9  - SAU
  5  2,9  - SOU
  5  3,1  - bAR
  5  3,1  - bER
  4  5,7  | AE
  4  5,7  | AI
  4  1,11 - AT
  4  6,7  - EAU
  4  2,10 - EAU
  4  5,7  | EE
  4  7,7  | EE
  4  5,7  | EO
  4  7,7  | EO
  4  1,11 - ET
  4  1,11 - IT
  4  5,7  | OE
  4  7,7  | OE
  4  5,7  | OI
  4  7,7  | OI
  4  2,9  - SAE
  4  2,9  - SAI
  4  2,9  - SAO
  4  2,9  - SEA
  4  2,9  - SEE
  4  2,9  - SEI
  4  2,9  - SIA
  4  2,9  - SIE
  4  2,9  - SOE
  3  2,9  - SI
  3  2,9  - SO
  2  6,6  - AA
  2  6,6  - AE
  2  5,7  - AE
  2  1,10 - AE
  2  6,6  - AI
  2  5,7  - EE
  2  6,7  - EE
  2  1,10 - EE
  2  2,10 - EE
  2  6,7  - EO
  2  2,10 - EO
  2  5,7  - OE
  2  1,10 - OE
