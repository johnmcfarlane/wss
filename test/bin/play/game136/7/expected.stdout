 39  1,9  | EAU
 32  4,4  | PIXIE
 29  1,6  | OHIA
 24  1,7  | HAE
 24  1,7  | HAO
 24  1,7  | HIE
 24  1,7  | HOE
 24  1,7  | HOI
 24  1,11 - UTA
 24  1,11 - UTE
 22  1,8  - IDEA
 22  1,8  - IDEE
 22  3,7  | JIAO
 22  3,7  | JOIE
 22  1,8  - ODEA
 21  1,11 - ETUI
 19  1,8  - ADO
 19  1,8  - IDE
 19  3,7  | JAI
 19  1,8  - ODA
 19  1,8  - ODE
 17  2,8  - DEE
 17  2,8  - DOE
 15  1,11 - ATE
 15  1,11 - ETA
 15  1,9  - USE
 14  1,9  | AE
 14  2,8  - DA
 14  2,8  - DE
 14  2,8  - DO
 14  1,9  | EE
 14  3,10 | EE
 14  1,9  | EO
 14  3,10 | EO
 14  3,7  | JA
 14  3,7  | JE
 14  3,7  | JO
 14  1,9  | OE
 14  3,10 | OE
 14  3,10 | OI
 12  1,9  - ESE
 12  1,9  - OSE
 12  2,11 - TAU
 12  2,11 - TOEA
 12  2,11 - TUI
 10  1,10 | AE
 10  1,6  | AHI
 10  1,10 | AI
 10  1,10 | EE
 10  1,10 | OE
 10  1,10 | OI
  9  2,11 - TAE
  9  2,11 - TAI
  9  2,11 - TAO
  9  2,11 - TEA
  9  2,11 - TEE
  9  2,11 - TIE
  9  2,11 - TOE
  9  1,9  - US
  8  3,9  | AE
  8  5,7  | EAU
  8  7,7  | EAU
  8  3,9  | OE
  8  6,5  - VAU
  7  1,8  - AD
  7  1,8  - ED
  7  1,7  | HA
  7  1,7  | HE
  7  1,7  | HI
  7  1,7  | HO
  7  1,8  - ID
  7  1,8  - OD
  7  1,11 - UT
  7  6,5  - VEE
  7  6,5  - VIA
  7  6,5  - VIE
  7  6,5  - VOE
  7  6,5  - VU
  6  1,6  - AE
  6  7,6  | AE
  6  1,6  | AH
  6  1,9  - AS
  6  1,6  - EE
  6  7,6  | EE
  6  1,6  | EH
  6  7,6  | EO
  6  1,9  - ES
  6  1,9  - IS
  6  1,6  - OE
  6  1,6  | OH
  6  1,9  - OS
  6  2,9  - SUE
  6  2,11 - TA
  6  2,11 - TE
  6  2,11 - TI
  6  2,11 - TO
  5  4,1  | AL
  5  2,9  - SAU
  5  2,9  - SOU
  5  3,1  - bAR
  4  5,7  | AE
  4  5,7  | AI
  4  1,11 - AT
  4  6,7  - EAU
  4  2,10 - EAU
  4  5,7  | EE
  4  7,7  | EE
  4  5,7  | EO
  4  7,7  | EO
  4  1,11 - ET
  4  1,11 - IT
  4  5,7  | OE
  4  7,7  | OE
  4  5,7  | OI
  4  7,7  | OI
  4  2,9  - SAE
  4  2,9  - SEA
  4  2,9  - SEE
  4  2,9  - SEI
  3  2,9  - SA
  3  2,9  - SI
  3  2,9  - SO
  2  6,6  - AA
  2  6,6  - AE
  2  5,7  - AE
  2  1,10 - AE
  2  6,6  - AI
  2  5,7  - EE
  2  6,7  - EE
  2  1,10 - EE
  2  2,10 - EE
  2  6,7  - EO
  2  2,10 - EO
  2  5,7  - OE
  2  1,10 - OE
